### DISCUSSION TITLE
Support for modded blocks

### DISCUSSION THREAD

**Discussion by zquillan** [mircokroon/minecraft-world-downloader#496] [General]
Serious question -
How much would it take per month to sponsor y'all to add support for modded blocks? Is this a feasible feature? I've been dying to chunk out some of the maps in a certain recreation server based on a high-profile fantasy book-series-turned-TV-show.

**Comment by mircokroon** [comment]
Hard to say, which Minecraft version and mod system (Forge/Fabric) are you looking for?

I'll probably get around to it _eventually_, if you _want to_ pay for it to happen sooner send me an email (end of readme) or leave your contact info and we can discuss privately.

**Reply by zquillan** [reply]
1.12.2 Forge, to be less ambiguous I'm looking to chunk out WesterosCraft for personal use, so a relatively high volume of modded blocks would be involved. I'll reach out, thanks.

**Reply by zquillan** [reply]
I shot you an email but gmail said it didn't go through - my address is skeletonguy8065@gmail.com if you wanna send a quick hello.

**Comment by hampusgrundel** [comment]
Is there any news about this?

**Comment by Mc5wagger** [comment]
I think every Minecrafter should have this mod. This would be a huge feature to really open up the doors to the rest of the Minecrafting community.

**Comment by Zhang-CT** [comment]
Is there any news about this?
